packages/ui/src/components/numpad/numpad-key.native.tsx

```diff
@@ -0,0 +1,50 @@
+import { ReactNode } from 'react';
+
+import { isString } from '@leather.io/utils';
+
+import { Box } from '../box/box.native';
+import { TouchableOpacity } from '../button/touchable-opacity.native';
+import { Text } from '../text/text.native';
+
+function normalizeKeyElement(element: ReactNode) {
```

Comment:
Maybe this can just be a react component - `KeyElement` with `HasChildren` type?

Reply:
Good call.  Honestly, for an component this small extracting this was an overkill to begin with. I'll inline this for now, as it's going to be removed in favor of SVGs anyway. 
I anticipate the **text or not** to be a recurring pattern though, so I'll look into making a more generic version of KeyElement as a helper component at some point.